lib/block.ml

```diff
@@ -160,14 +160,20 @@ let rec error_padding = function
       let xs = error_padding xs in
       x :: xs
 
-let pp_errors ppf t =
+let pp_errors ?syntax ppf t =
   match t.value with
   | OCaml { errors = []; _ } -> ()
-  | OCaml { errors; _ } ->
-      let errors = error_padding errors in
-      Fmt.pf ppf "```mdx-error\n%a\n```\n"
-        Fmt.(list ~sep:(any "\n") Output.pp)
-        errors
+  | OCaml { errors; _ } -> (
+      match syntax with
+      | Some Syntax.Mli | Some Syntax.Mld ->
+          let errors = error_padding errors in
+          Fmt.pf ppf "][\n%a\n" Fmt.(list ~sep:(any "\n") Output.pp) errors
+      | Some Syntax.Cram -> ()
```

Comment:
i think some syntax for cram output should be invented (or at least keep the markdown syntax to not integrate regression!)